### PR TITLE
Add basic Log4J Logging

### PR DESCRIPTION
**Pull request by grimreaper**
This is really here to add logback.xml and to show how it gets called.

example logs:
image

### CODE REVIEW

File: code/src/java/pcgen/system/Main.java [hunk 1]
```diff
@@ -129,6 +134,19 @@ private static void logSystemProps()
 	}
 
 	/**
+	 * Don't delete. Used for testing logging.
```
Comment (thpr) [inline code/src/java/pcgen/system/Main.java]: Why have a test in main line code rather than out in test code?

Comment (grimreaper) [inline code/src/java/pcgen/system/Main.java]: this is mainly there so I could add a line to main() and run. It was very useful for testing that the classpath was set up correctly. This differs from main to test. I don't /really/ need this function. It was just nice to have.

**Comment by grimreaper** [comment]
I'm mainly concerned about if you like the style of how to get the logger. If that's okay the rest is easy to change.

**Comment by cpmeister** [comment]
I don't think that slf4j can be used in PCGen because PCGen uses some custom logging levels that slf4j cannot provide. I think that using Log4j directly is a better approach.

**Comment by grimreaper** [comment]
What kind of custom logging levels? They are typically better suited as adding contextual information? I've seen CONFIG? anything else?

**Comment by cpmeister** [comment]
Take a look at pcgen.util.PCGenLogLevel

**Pull request merged**